src/components/Button.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { Button } from "./Button";
+
+// More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
+export default {
+  title: "Swapr-ui/Button",
```

Comment:
Small think to bear in mind in future:
This title path groups the components, so if we keep pushing `Swapr-ui/Button`, there will be a big `Swapr-ui` folder in Storybook

IMHO this doesn't make much sense as we're in this repo. I'd use this grouping feature only if we have component groups either by functionality or design (for example atoms, molecules, etc).

Reply:
Makes sense, but for now, It's just to show case, so I'll keep it like it is. But I think you have experience in that and you can lead the storybook to put it neat. 🙌